hub/powertoys/text-extractor.md

```diff
@@ -31,3 +37,65 @@ From the Settings menu, the following options can be configured:
 | Setting | Description |
 | :--- | :--- |
 | Activation shortcut | The customizable keyboard command to turn on or off always on top for that window. |
+
+## Supported languages
+
+Text Extractor can only recognize languages that have the OCR pack installed.
+
+The list can be obtained via PowerShell by running following commands:
+```console
+PS C:\Windows\system32> [Windows.Media.Ocr.OcrEngine, Windows.Foundation, ContentType = WindowsRuntime]
+PS C:\Windows\system32> [Windows.Media.Ocr.OcrEngine]::AvailableRecognizerLanguages
+```
+
+### OCR packs installation
+
+For supported languages, the OCR language pack can be installed along with the language via <kbd>Language</kbd>-><kbd>Preferred languages</kbd>-><kbd>Add language menu</kbd>.
+
+It's possible to get a list of all supported languages by running this command in PowerShell in Administrator mode:
+```console
+PS C:\Windows\system32> Get-WindowsCapability -Online | Where-Object { $_.Name -Like 'Language.OCR*' }
+```
+
+Example output:
+```console
+....
+Name  : Language.OCR~~~el-GR~0.0.1.0
+State : NotPresent
+
+Name  : Language.OCR~~~en-GB~0.0.1.0
+State : NotPresent
+
+Name  : Language.OCR~~~en-US~0.0.1.0
+State : Installed
+
+Name  : Language.OCR~~~es-ES~0.0.1.0
+State : NotPresent
+
+Name  : Language.OCR~~~es-MX~0.0.1.0
+State : NotPresent
+....
+```
+
+If a language is not available in the output, then it's not supported by OCR.
+
+If the state is <kbd>NotPresent</kbd>, then it's possible to install OCR pack for the language.
+
+For example, the following commands install the OCR pack for <kbd>en-US</kbd> in PowerShell run as Administrator:
+```console
+PS C:\Windows\system32> $Capability = Get-WindowsCapability -Online | Where-Object { $_.Name -Like 'Language.OCR*en-US*' }
+PS C:\Windows\system32> $Capability | Add-WindowsCapability -Online
+```
+These commands remove it:
+```console
+PS C:\Windows\system32> $Capability = Get-WindowsCapability -Online | Where-Object { $_.Name -Like 'Language.OCR*en-US*' }
+PS C:\Windows\system32> $Capability | Remove-WindowsCapability -Online
+```
+
+## Troubleshooting
```

Comment:
how does Install a language pack needs to be a section

Reply:
Have removed "### OCR packs installation" line. I guess that's what you meant, right?

Reply:
no, if i don't have the OCR pack needed, we should say how they get it.  "Hey, i want german, ..."

Reply:
Hi Clint,

I've made changes according to your comments.
Could you please check whether it looks good to you?

Thank you,
Taras

Reply:
Looks good! Thank you Taras!